editor/src/clj/editor/workspace.clj

```diff
@@ -456,20 +457,20 @@ ordinary paths."
 (defn- unpack-editor-plugins! [workspace changed]
   ; Used for unpacking the .jar files and shared libraries (.so, .dylib, .dll) to disc
   ; TODO: Handle removed plugins (e.g. a dependency was removed)
-  (let [changed-resources (set (map resource/proj-path changed))
-        all-plugin-resources (find-plugins-shared-libraries workspace)
-        changed-plugin-resources (filter (fn [x] (contains? changed-resources (resource/proj-path x))) all-plugin-resources)
-        changed-shared-library-resources (filter is-shared-library? changed-plugin-resources)
-        changed-jar-resources (filter is-jar-file? changed-plugin-resources)]
+  (let [changed-plugin-resources (into [] (filter #(is-plugin-file? workspace %)) changed)]
```

Comment:
Similar performance improvement. The previous implementation went through all project resources, filtered them for plugin files, then filtered once more to only include changed files. There was no need to get all files since filtering changed files for plugin files is enough.

Reply:
Yeah good point, I'll fix the `is-plugin-file?` fn!